packages/components/src/utils/config-values.js

```diff
@@ -54,6 +54,10 @@ export default {
 	cardPaddingX: CARD_PADDING_X,
 	cardPaddingY: CARD_PADDING_Y,
 	cardPadding: `${ CARD_PADDING_X } ${ CARD_PADDING_Y }`,
+	cardPaddingXSmall: `${ space( 2 ) }`,
+	cardPaddingSmall: `${ space( 4 ) }`,
+	cardPaddingMedium: `${ space( 4 ) } ${ space( 6 ) }`,
+	cardPaddingLarge: `${ space( 6 ) } ${ space( 8 ) }`,
```

Comment:
All of the numbers have doubled compared to the previous settings (see the `bodySizes()` and `headerFooterSizes()` functions in the deleted code of the `packages/components/src/card/styles.js` file), because the `space()`  function used in this config file is based on a grid with a base unit that is half of the previous grid unit.

Reply:
This makes a lot of sense! I wonder if we can make it even more obvious by having something like a `basePadding` that would be `2` and then have all the defaults factor that into the default value, so it would be even more obvious as to where they come from and how they're calculated.

Reply:
Where would you add this `basePadding` variable?

Previously these padding values were defined directly in the `styles` file of the `Card` component.
Now, we're moving these padding values to a common configuration file. In this file, the `space` function is different from the one that was used by the old `Card` component, and so I'm not sure it makes much sense to have a `basePadding` variable here.

I guess that when we migrate all components to g2, these components would all use this configuration file — and inherently this `space` function.

Reply:
That's super minor, feel free to disregard it. It could add unnecessary complexity to something that will likely change in the future, so I'm fine to leave it as-is.